src/main/webapp/index.html

```diff
@@ -48,8 +50,23 @@ <h3>Want to view a recommendation?</h3>
         <button class="map-category-button">Study Places</button>
       </div>
     </section>
-    <!-- The Recommendation Map -->
-    <div id="map"></div>
+    <div class="wrapper">
+      <!-- The Recommendation Map -->
+      <div id="map"></div>  
+      <!-- View Recommendation Area -->
+      <div id="rec-container">
+        <h3>Sky Garden</h3>
+        <h4>Location / Address</h4>
+        <p id="rec-address">51.5109° N, 0.0837° W</p>
+        <h4>Description</h4>
+        <p>Sky garden is a great place to visit, with really good
+          views of London. You can enjoy the 360 degree view
+          of the city - for free! They are public gardens and 
+          definitely worth a visit when you have the time.
+          Remember to keep socially distanced!
+        </p>
+      </div>
+    </div>
```

Comment:
Based on these changes I think this branch is based on the rec-map-markers branch, hence these are repeated.
Can you please edit the pull request or maybe rebase if necessary, thanks! :D